### PR TITLE
Add Release Card Component

### PR DESCRIPTION
**Pull request by tameremad**
image
image

### CODE REVIEW

File: services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            MDComponent.current = new MDCRipple(control.current);
+        }
+        return () => MDComponent.current?.destroy();
+    }, []);
+
+    return (
+        <div className={classNames('mdc-card release-card', className)}>
+            <div className="release-card__header">
+                <div className="release__title mdc-typography--headline6">{title}</div>
+                <Button className="release__hash" label={hash} />
```
Comment (hannesg) [inline services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx:48]: The hash can be empty. We should probably hide this button then.

File: services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        if (control.current) {
+            MDComponent.current = new MDCRipple(control.current);
+        }
+        return () => MDComponent.current?.destroy();
+    }, []);
+
+    return (
+        <div className={classNames('mdc-card release-card', className)}>
+            <div className="release-card__header">
+                <div className="release__title mdc-typography--headline6">{title}</div>
```
Comment (hannesg) [inline services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx:47]: What will we do when the title is very long?

**Review by hannesg — APPROVED** [review]

File: services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx (hunk 1 above)
Comment (tameremad) [reply services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx:48]: I'm curious when this could happen? 
for the undeploy versions?

**Review by Medo-fdc — APPROVED** [review]
I think we should move ReleaseCard directory inside Components directory

File: services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx (hunk 2 above)
Comment (tameremad) [reply services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx:47]: image

Comment (medo-freiheit) [reply services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx]: try just to add tails and only one row e.g ( something something ...)

Comment (tameremad) [reply services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx]: I think it's important for devs to see the entire title.

**Pull request merged**